packages/react/src/components/table/Table.stories.tsx

```diff
@@ -0,0 +1,919 @@
+import React, { useState } from 'react';
+import { parse, isBefore, isSameDay } from 'date-fns';
+
+import { Table } from './Table';
+import workTrial from './story-example-work-trial.json';
+import { Button } from '../button';
+import { IconTrash } from '../../icons';
+
+export default {
+  component: Table,
+  title: 'Components/Table',
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {},
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const Default = (args) => {
+  const cols = [
+    { key: 'id', headerName: 'Not rendered' },
+    { key: 'firstName', headerName: 'First name' },
+    { key: 'surname', headerName: 'Surname' },
+    {
+      key: 'age',
+      headerName: 'Age',
+      transform: ({ age }) => {
+        return <div style={{ textAlign: 'right' }}>{age}</div>;
+      },
+    },
+    { key: 'profession', headerName: 'Profession' },
+  ];
+
+  const rows: Array<object> = [
+    { id: 1000, firstName: 'Lauri', surname: 'Kekkonen', age: 39, profession: 'Engineer' },
+    { id: 1001, firstName: 'Maria', surname: 'Sarasoja', age: 62, profession: 'Designer' },
+    { id: 1002, firstName: 'Anneli', surname: 'Routa', age: 50, profession: 'Meteorologist' },
+    { id: 1003, firstName: 'Osku', surname: 'Rausku', age: 18, profession: 'Mail Carrier' },
+  ];
+
+  const caption = (
+    <span>
+      <b>Table 1</b>: Table description
+    </span>
+  );
+
+  return (
+    <div style={{ maxWidth: '640px' }}>
+      <Table cols={cols} rows={rows} caption={caption} indexKey="id" renderIndexCol={false} />
+    </div>
+  );
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const Dark = (args) => {
```

Comment:
Yes they could. Should we go with name 'Dark' or 'Default'? The default variant is the dark variant. In Core version there is no default at the moment. It is coded so that you need to tell which variant you want to use. It could have probably been coded differently there, but it felt more clear to give them separate classnames: https://hds.hel.fi/storybook/core/?path=/info/components-table--dark

Reply:
Default is ok 👍

Reply:
- [ ] Combine default and dark: remove dark